docs/FRSKY_R9.md

```diff
@@ -21,31 +21,45 @@ The Frsky R9M transmitter module and R9MX receiver are commercially available an
   </tbody>
 </table>
 
-Comment: mLRS also supports the R9MM receiver. However, flashing the R9MM with ST-Link is really tedious and requires top soldering skills or employing other tricks, as one needs to connect to four tiny solder pads. You really should consider using the R9MX receiver instead. Note: The R9MM has a different wiring diagram than the R9MX.
-
 ## R9M Tx Module ##
 
-The R9M module is unfortunately limited with respect to serial ports. It provides acess to only one serial port, which moreover has inverted TTL signals. In order to use this serial port, these conditions apply:
+The R9M module is somewhat limited with respect to serial ports. It provides access to only one serial port, which moreover has inverted TTL signals. In order to use this serial port, these conditions apply:
+
+1. One needs to use a "Frsky inverter" dongle. You either can buy one or build it yourself.  Alternatively, if you will use the Lua script for configuration, and wish to use the serial port to connect via MAVLink wirelessly to a Ground Control Station, you can use the one of the supported esp32 boards with the mlrs-wifi-bridge sketch.
 
-1. One needs to use a "Frsky inverter" dongle. You either can buy one or build it yourself.
+2. The serial port of the R9M module can be configured to work either as "serial" or "CLI". This is done by setting the dip switch 1 (left dip switch): off (switch down) = CLI, on (switch up) = serial. Note that the dip switch position is read only at power up, i.e., one needs to re-power the module to make any change effective.
 
-2. The serial port of the R9M module can be configured to work either as "serial" or "CLI". This is done by setting the dip switch 1 (left dip switch): off (switch down) = CLI, on (switch up) = serial. Note that the dip switch position is read only at power up, i.e., one needs to repower the module to make any change effective.
+3. If you have the older 2018 "ACCST" version of the R9M, you will want to perform the Inverter Mod to allow reliable communication with the radio at higher bit rates.  This is nicely documented in the [ExpressLRS documentation](https://www.expresslrs.org/hardware/inverter-mod/).
```

Comment:
Maybe one wants to add the "newer" alternative. Like "This mod is not needed for the newer "XX what was it, can't recall XX" version (can be identified by a XXXXX label printed to the front)", or so.